aws/backoff_retry_lambda/backoff_retry.tf

```diff
@@ -37,5 +36,6 @@ resource "aws_lambda_event_source_mapping" "dead_letters" {
 
 resource "aws_cloudwatch_log_group" "backoff_log_group" {
   name              = "/aws/lambda/${aws_lambda_function.backoff_retry.function_name}"
+  kms_key_id        = data.aws_kms_key.cloudwatch.arn
```

Comment:
We have no issues with managed KMS keys for this.

Reply:
Done and done - reverted to default CloudWatch service key.